### ISSUE TITLE
pipenv not included?

### ISSUE THREAD

**Issue by ranomier** [flathub/com.jetbrains.PyCharm-Community#50]
It seems like pipenv is not included, but it is choosable from the new project wizard.

![image](https://user-images.githubusercontent.com/7649949/52114539-94d59880-260c-11e9-8471-72a5bb8a42a4.png)
![image](https://user-images.githubusercontent.com/7649949/52114620-cc444500-260c-11e9-9621-e9f1db58c686.png)![image](https://user-images.githubusercontent.com/7649949/52114768-3bba3480-260d-11e9-906b-9f8f6283dc19.png)

Should i make a pull request?

Seems pretty simple to add it to "com.jetbrains.PyCharm-Community.json"

**Comment by ranomier** [comment]
Maybe like this?

```
        {
        "name": "python3-pipenv",
        "buildsystem": "simple",
        "build-commands": [
            "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} pipenv"
        ],
        "sources": [
            {
                "type": "file",
                "url": "https://files.pythonhosted.org/packages/fd/e9/01822318551caa0d62a181ba3b10f0f3757bb1e270da97165bd52db92776/pipenv-2018.11.26.tar.gz",
                "sha256": "a673e606e8452185e9817a987572b55360f4d28b50831ef3b42ac3cab3fee846"
            }
        ]
},
```

**Comment by FakeShemp** [comment]
Feel free to make a pull request. I take all help I can 🙂

**Comment by ranomier** [comment]
ok, first i have to learn how :).

But i will make it today.

**Comment by ranomier** [comment]
So i made the mentioned changes to https://github.com/ranomier/com.jetbrains.PyCharm-Community
I tried to build it but i got this error:

```
[...]
  Collecting setuptools
    Could not find a version that satisfies the requirement setuptools (from versions: )
  No matching distribution found for setuptools
  
  ----------------------------------------
Command "/usr/bin/python3.7 -m pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-ov2vz8wl --no-warn-script-location --no-binary :none: --only-binary :none: --no-index --find-links file:///run/build/python3-pipenv -- setuptools wheel" failed with error code 1 in None
Error: module python3-pipenv: Child process exited with code 1
```

Maybe you have an idea. What i understand from google is, that maybe pip is too old.

**Comment by ranomier** [comment]
I think i am at the bottom of the problem.

setup.py has a key, that needs internet connection. Regardless what pip says. See here:
https://github.com/pypa/pip/issues/410#issuecomment-3186621

This could be the solution, i now have to find the right directory to put the file in:
https://pip.pypa.io/en/latest/reference/pip_install/#controlling-setup-requires
This could help finding the right directory:
https://docs.python.org/3/install/index.html#distutils-configuration-files

maybe later. Now i need a brake. Pyhton packaging is hell.

**Comment by ranomier** [comment]
@FakeShemp Could you look into the pull request please?

**Comment by FakeShemp** [comment]
Solved by #51
